spring-boot/admin/src/test/java/eu/coatrack/admin/YggAdminApplicationTests.java

```diff
@@ -22,11 +22,11 @@
 
 import org.junit.jupiter.api.Test;
 import org.springframework.boot.autoconfigure.EnableAutoConfiguration;
-import org.springframework.boot.autoconfigure.security.oauth2.OAuth2AutoConfiguration;
 import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.boot.test.mock.mockito.MockBean;
+import org.springframework.security.oauth2.client.OAuth2AuthorizedClientService;
```

Comment:
Why add new imports to the class if you do not add new code?

Reply:
Once again, this might have been some tests that I was doing, and forgot to then remove the imports/optimize imports.
Sorry for that, so I removed and the YggAdminApplicationTests is working well, as seen in the screenshot below:

![image](https://user-images.githubusercontent.com/47450617/185879574-0e797dbc-328a-4c8e-a032-859ff2104147.png)